src/gsi/gsimod.F90

```diff
@@ -1001,6 +1035,46 @@ module gsimod
   namelist/superob_radar/del_azimuth,del_elev,del_range,del_time,&
        elev_angle_max,minnum,range_max,l2superob_only,radar_sites,radar_box,radar_rmesh,radar_zmesh
 
+! RADARUSE_directDA
+!     mphyopt          - microphysics scheme to use in the forward operator
+!                        (2-6(LIN) and 108(TM) are supported for now)
+!     l_set_be_rw      - re-set background error statistics for using radar wind
+!                        obs (.TRUE.: on  ; .FALSE.: off)
+!     l_set_be_dbz     - re-set background error statistics for using radar dbz
+!                        obs (.TRUE.: on  ; .FALSE.: off)
+!     be_sf            - multiplying factor to tune the background error
+!                        standard deviation of stream function
+!     hscl_vf          - horizontal background error correlation length scale of
+!                        stream function
+!     hscl_vf          - vertical   background error correlation length scale of
+!                        stream function
+!     l_decouple_sf_vp - de-couple the correlation/balance between s.f. and v.p.
+!     l_decouple_sf_tps- de-couple the correlation/balance between s.f. and temperature, ps
+!     rw_obs4wrd_bmwth - beam width impact on radar wind obs forward operator
+                         ! 1: GSI original (vrminmax)
+                         ! 2: simple vertical interpolation
+                         ! 3: weighted average of multiple-layers
+!     l_dbz4wrd_melt   - effect of melting snow in dbz obs forward operator
+!     l_use_cvpqx     - use power transform option to qx (qr/qs/qg)
+!     for more information, please refer to directDA_radaruse_mod.f90
+!
+  namelist/radaruse_directDA/mphyopt,oe_rw,oe_dbz,l_set_be_rw,l_set_be_dbz,   &
```

Comment:
Some parameters in this namelist have no explanation in comment black. Please try to add explanation for all parameters.

Reply:
I added explanation of parameters in comment block according to your comments.

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-fa08cb92718d118dfe569a760912124e0d84ebdddad08003381497e6c4407a92R1036-R1232